components/TextFields/src/ColorThemer/MDCTextFieldColorThemer.h

```diff
@@ -72,7 +75,8 @@
  using the default color class properties.
 
  @warning This method will soon be deprecated. Consider using
- +applySemanticColorScheme:toAllTextInputControllersOfClass: instead.
+ +applySemanticColorScheme:toAllTextInputControllersOfClass: instead. Learn more at
```

Comment:
They should be using the outlined/filled color themers instead.

Reply:
I'm not sure what change needs to be made here - will leave as a follow-up.